File: src/ApproximatorTester/ApproximatorTester.Tests/test/data/Js/FourVars.js
{caret}function main(cond)
{
    var query1 = "Select";
    var field = " fieldName ";
    query1 += "from";
    var tableName = "Table1";
    if (cond)
    {
        query1 += tableName;
    }
    else
    {
        tableName = "Table2";
        query1 += tableName;
    }
    var filter = "where field is numeric";
    query1 += filter;
    execScript(query1);
}

File: src/ApproximatorTester/ApproximatorTester.Tests/test/data/CSharp/FourVars.cs
{caret}namespace N
{
    class Program
    {
        static void Main(string[] args)
        {
            string query1 = "a";
            const string field = " not used ";
            query1 += "b";
            var tableName = "c";
            if (args.Length != 0)
            {
                query1 += tableName;
            }
            else
            {
                tableName = "d";
                query1 += tableName;
            }
            string filter = "e";
            query1 += filter;
            Program.ExecuteImmediate(query1);
        }

        static void ExecuteImmediate(string query)
        {
            
        }
    }
}
